CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_shopify v0.11.0
+[PR #74](https://github.com/fivetran/dbt_shopify/pull/74) includes the following updates: 
+
+## 🚨 Breaking Changes 🚨
+- Added `source_relation` to the `partition_by` clauses that determine the `customer_index` in the `int_shopify__customer_email_rollup` table. If the user is leveraging the union feature, this could change data values, so would recommend a `dbt run --full-refresh` in this case. 
```

Comment:
Same note from the source, let's remove the bit about the full refresh as it is not necessary for these models.
```suggestion
- Added `source_relation` to the `partition_by` clauses that determine the `customer_index` in the `int_shopify__customer_email_rollup` table. If the user is leveraging the union feature, this could change data values. 
```